packages/application-extension/src/index.ts

```diff
@@ -393,6 +394,35 @@ const shell: JupyterFrontEndPlugin<INotebookShell> = {
   provides: INotebookShell,
 };
 
+/**
+ * The default splash screen provider.
+ */
+const splash: JupyterFrontEndPlugin<ISplashScreen> = {
+  id: '@jupyterlab/apputils-extension:splash',
```

Comment:
For consistency with the other plugins:

```suggestion
  id: '@jupyter-notebook/application-extension:splash',
```